src/public/js/likeToggle.js

```diff
@@ -0,0 +1,34 @@
+    document.addEventListener('DOMContentLoaded', function () {
+    const likeForms = document.querySelectorAll('.like-form');
+    const unlikeForms = document.querySelectorAll('.unlike-form');
+
+    likeForms.forEach(function (likeForm) {
+        likeForm.addEventListener('submit', function (e) {
+            e.preventDefault(); // Prevent the default form submission
+            likeForm.querySelector('.like').classList.add('d-none');
+            likeForm.nextElementSibling.querySelector('.unlike').classList.remove('d-none');
+
+            fetch(likeForm.action, {
+                method: 'POST',
+                headers: {
+                    'X-CSRF-TOKEN': likeForm.querySelector('input[name="_token"]').value,
```

Comment:
???

Reply:
Goodafternoon po sir, any context po for this comment? Thank you po. Hindi ko po alam if ano po gagawain 🙇🙇